jsonschema/compat.py

```diff
@@ -40,6 +43,9 @@ def urlsplit(url):
     return SplitResult(scheme, netloc, path, query, fragment)
 
 
+DefragResult = namedtuple('DefragResult', 'url fragment')
```

Comment:
That is a good point about tuple comparison, I hadn't considered that.

I think I will remove this. It's only used locally in a single function now, so the attributes are a lot less necessary.